packages/react-theme/src/stories/ThemeColors.main.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { teamsDarkTheme, teamsHighContrastTheme, teamsLightTheme, webDarkTheme } from '../index';
 import { Theme } from '../types';
-import { ColorRampItem } from './ColorRamp.stories';
+import { ColorRampItem } from './ColorRamp.main.stories';
```

Comment:
Theme stories are a bit messy with render functions shared between stories. Cleanup for another time.